### PR TITLE
Showcase feature right after the toolbar header

### PR DESCRIPTION
**Pull request by bernyhas**
Add Showcase Feature headline section right after the toolbar Navigation portfolio

### CODE REVIEW

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @bernyhas , 

Great work on making the changes  👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️

- [ ] Kindly remove the bullet points from the `li` 's
- [ ] Kindly style the icons such that they are placed horizontally, you have the option of flexbox to achieve this.

### Suggestions

_Please make sure that pull requests are approved before closing them if for any reason you close it before approval, explain why it was closed in the new pull request_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by bernyhas** [comment]
Hi @AshakaE ,

As per your requested  bullet point from the list style has remove it  and flexbox has been used to align the icon properly.
just waiting for your approval.

Thanks
Hans

File: index.html [hunk 1]
```diff
@@ -20,7 +20,40 @@
           <h1>Hans Bernadin</h1>
```
Comment (hemantso) [inline index.html]: - [ ] Kindly add the logo in a `<a>` tag. It will help the user to navigate.

File: index.html [hunk 2]
```diff
@@ -20,7 +20,40 @@
...
+            I'm Hans <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3>Let's Connect</h3>
```
Comment (hemantso) [inline index.html]: - [ ] Kindly add the `Let's Connect`  in a `<a>` tag. It will help the user to navigate.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @bernyhas , 

Good job so far! 👍 
You still need to work on some issues to go to the next project, but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._
- [ ] Fix the logo
- [ ] Fix the let's connect heading

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please update the `live link` in the README file

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bernyhas** [comment]
Hi @hemant-soni-vst-au4  as per your request update the html  a tag to the h3 tag and ready to move on.

Thanks

**Comment by bernyhas** [comment]
Hi reviewer i have updated the project you have requested but you still not replied to complete the project please see my comment complete that in order to submit the project. Thanks

Sent from my iPhone

On Jul 8, 2021, at 07:07, Hemant soni ***@***.***> wrote:

﻿

@hemant-soni-vst-au4 requested changes on this pull request.

Hi <STUDENT_MENTION>,

Good job so far! 👍
You still need to work on some issues to go to the next project, but you are almost there!

[Alt Text]<https://camo.githubusercontent.com/aefcc619097caf00ee032a9910c2df54fb144addae34ff1c7d30b063e506085b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f713655346662726543344e656f2f67697068792e676966>

Required Changes ♻️

Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

  *   Please update the live link in the README file

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me ***@***.***<https://github.com/hemant-soni-vst-au4>) so I can see your comments and respond.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

________________________________

As described in the Code reviews limits policy<https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form<https://airtable.com/shrQAqnBwek5a0O0s>.

________________________________

In index.html<https://github.com/bernyhas/PortFolio/pull/5#discussion_r666092212>:

> @@ -20,7 +20,40 @@
           <h1>Hans Bernadin</h1>


  *    Kindly add the logo in a <a> tag. It will help the user to navigate.

________________________________

In index.html<https://github.com/bernyhas/PortFolio/pull/5#discussion_r666092641>:

> @@ -20,7 +20,40 @@
           <h1>Hans Bernadin</h1>
           <a href="#"><i class="fas fa-bars"></i></a>
         </nav>
+        <section class="showcase">
+          <h2>
+            I'm Hans <br />
+            Glad to see you!
+          </h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3>Let's Connect</h3>


  *    Kindly add the Let's Connect in a <a> tag. It will help the user to navigate.

**Comment by bernyhas** [comment]
Hi @hemant-soni-vst-au4 as per your request update the html a tag to the h3 tag and ready to be approve.

Thanks

**Comment by nidalaa** [comment]
Hello @bernyhas ,

_This is an additional review as a result of the second opinion request._


In general, the confusion here is caused by you opening 2 pull requests for the same project @bernyhas.

You got your first review here https://github.com/bernyhas/PortFolio/pull/3#pullrequestreview-701837488, then @AshakaE made another review that looks like the first review in the current pull request.

When @hemant-soni-vst-au4 came here everything looked like the second review, while it was the final review :(

As @hemant-soni-vst-au4 was asking for additional changes, not the changes asked by @AshakaE in the previous review, we are going to approve this project.

However, @bernyhas - always remember to use the same pull request for the entire project to avoid confusion (and if it is totally impossible - let your reviewer know where they can find the previous review in your PR description). 
Not doing that is not fair - as Code Reviewers cannot do their job in the best way and you might abuse code review limits.

Your project will appear as approved in your Dashboard in ~1h.

Happy coding 🚀!

**Pull request merged**